app/controllers/vehicles_controller.rb

```diff
@@ -0,0 +1,9 @@
+class VehiclesController < ApplicationController
+  def brands
+    render_response(Vehicles::BrandService.new.call)
```

Comment:
i do not understand what is that service doing
if you want to return a list of car brands, name service `Vehicles::BrandListService`
also, you should only call `.perform` method on service